tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/types/TString.java

```diff
@@ -235,23 +232,26 @@ public TString using(Charset charset) {
 
   static <T> Tensor<TString> createTensor(NdArray<T> src, Function<T, byte[]> getBytes) {
     long size = StringTensorBuffer.computeSize(src, getBytes);
-    return Tensor.of(TString.DTYPE, src.shape(), size, data ->
-        ((TStringImpl)data).tensorBuffer.init(src, getBytes)
-    );
+    return Tensor.of(
+        TString.DTYPE,
+        src.shape(),
+        size,
+        data -> ((TStringImpl) data).tensorBuffer.init(src, getBytes));
   }
 
   static TString mapTensor(TF_Tensor nativeTensor, Shape shape) {
     StringTensorBuffer buffer = TensorBuffers.toStrings(nativeTensor, shape.size());
     return new TStringImpl(buffer, UTF_8_LAYOUT, shape);
   }
 
-  private static DataLayout<DataBuffer<byte[]>, String> UTF_8_LAYOUT = DataLayouts.ofStrings(StandardCharsets.UTF_8);
+  private static DataLayout<DataBuffer<byte[]>, String> UTF_8_LAYOUT =
+      DataLayouts.ofStrings(StandardCharsets.UTF_8);
 
   private final StringTensorBuffer tensorBuffer;
 
-  private TStringImpl(StringTensorBuffer buffer, DataLayout<DataBuffer<byte[]>, String> layout, Shape shape) {
+  private TStringImpl(
+      StringTensorBuffer buffer, DataLayout<DataBuffer<byte[]>, String> layout, Shape shape) {
     super(layout.applyTo(buffer), shape);
     tensorBuffer = buffer;
   }
 }
-
```

Comment:
I get a feeling that auto-formatter was a bit aggressive here, can you please just double-checked that the Google one was used when reformatting all the `types` classes?

Reply:
I have verified that google-java-format settings are installed and enabled in InteliJ

Reply:
There was a format change on TString.java and TBool.java when I double checked all the formats on the  types.

Reply:
Ok, it's fine I just wanted to make sure that the right formatter was applied